features/steps/assign_incomplete_case.py

```diff
@@ -0,0 +1,76 @@
+from behave import *
+from selenium.webdriver.common.action_chains import ActionChains
+from create_eligible_finances import create_eligible_finance
+
+
+@given(u'case notes are empty')
+def step_impl(context):
+    notes = context.helperfunc.find_by_name('case.notes')
+    assert len(notes.text) == 0
+
+
+@step(u'I have created a user')
+def step_impl(context):
+    context.execute_steps(u'''
+        When I select 'Create new user'
+        And enter the client's personal details
+        And I click the save button on the screen
+    ''')
+
+
+@step(u'I have created a valid discrimination scope')
+def step_impl(context):
+    context.execute_steps(u'''
+        When I select ‘Create Scope Diagnosis'
+        And I select the categories Discrimination, Direct Discrimination, Disability, Work
+        Then I get an INSCOPE decision
+        And select 'Create financial assessment'
+    ''')
+
+
+@given(u'I am on the Diversity tab')
```

Comment:
Could we change this to `when` so it fits in with the structure better?

Reply:
it could be changed to 'step' but its currently its set to given as thats how it is in the scenario, and simply reads better like this.